spec/helpers/foods_helper_spec.rb

```diff
@@ -0,0 +1,16 @@
+# require 'rails_helper'
+
+# # Specs in this file have access to a helper object that includes
+# # the FoodsHelper. For example:
+# #
+# # describe FoodsHelper do
+# #   describe "string concat" do
+# #     it "concats two strings with spaces" do
+# #       expect(helper.concat_strings("this","that")).to eq("this that")
+# #     end
+# #   end
+# # end
+# RSpec.describe FoodsHelper, type: :helper do
+#   pending "add some examples to (or delete) #{__FILE__}"
+# end
+# #
```

Comment:
- [ ] Also for all those files you commented the code please kindly delete them to keep your project clean 🙃

Reply:
Done!